src/components/VersionSelect/index.tsx

```diff
@@ -64,24 +67,31 @@ class VersionSelectBase extends React.Component<PublicProps> {
 
         <Form.Group as={Col} className={className}>
           <Form.Label>{label}</Form.Label>
-          <Form.Control as="select" value={value} onChange={this.onChange}>
-            {listedVersions.length && (
-              <optgroup
-                className={styles.listedGroup}
-                label={gettext('Listed')}
-              >
-                {listedVersions.map(this.renderOption)}
-              </optgroup>
-            )}
-            {unlistedVersions.length && (
-              <optgroup
-                className={styles.unlistedGroup}
-                label={gettext('Unlisted')}
-              >
-                {unlistedVersions.map(this.renderOption)}
-              </optgroup>
-            )}
-          </Form.Control>
+
+          {isLoading ? (
+            <div className="form-control">
+              <Skeleton />
```

Comment:
It would be nice to give this a specific width to mimic a version number (like `1.5.1`). We should strive to make skeleton screens as close to the final layout as possible. In this case, it would be easy to predict the average width of a version number. They will most likely be in the range of `0.1` to `99.0.0`.

Reply:
I did not set a width because the select also has an arrow on the right, so rendering the skeleton full width covers both.